APIs_Drivers/UnbufferedSerial/main.cpp

```diff
@@ -0,0 +1,35 @@
+#include "mbed.h"
```

Comment:
Why are copyrights missing in most code snippets? @AnotherButler

Reply:
@jamesbeyond

Reply:
traditionally we don't include copyrights in code snippets,
e.g. the snippets on this page: https://os.mbed.com/docs/mbed-os/v5.15/apis/callback.html
but now all the code snippets are in this folder and they are individual files, I am not sure whether they need to have license or not? if the full license it too long is there any short versions?
I'll let @0xc0170 @adbridge comment on this

Reply:
I would add there:

```
```

Reply:
[Done](https://github.com/ARMmbed/mbed-os-examples-docs_only/compare/dd81220d2f1c723333609afa51c1479d12251021..17b7244223e0208efdc12274754ea22467bc8234).